src/selectors/prevSibling.js

```diff
@@ -1,34 +1,50 @@
-import { getSortPreference, getThoughtsRanked, getThoughtsSorted, hasChild } from '../selectors'
-import { isFunction } from '../util'
+import { getContextsSortedAndRanked, getSortPreference, getThoughtsRanked, getThoughtsSorted, hasChild, isContextViewActive } from '../selectors'
+import { head, isFunction, perma } from '../util'
 
 /**
  * Gets a context's previous sibling with its rank.
  *
  * @param context   Can be a context or path.
  */
 export default (state, value, context, rank) => {
+
   const { showHiddenThoughts } = state
   const sortPreference = getSortPreference(state, context)
-  const siblings = (sortPreference === 'Alphabetical' ? getThoughtsSorted : getThoughtsRanked)(state, context)
+
+  const contextViewActive = perma(() => isContextViewActive(state, context))
```

Comment:
Since `contextViewActive` will always be used, you can use a normal variable:

```js
const contextViewActive = isContextViewActive(state, context)
```

`perma` is only needed if there is at least one case where it does not get used it all, in which case the function wrapper emulates a lazy-evaluated variable.

Reply:
That makes sense